tests/test_neighborhood.py

```diff
@@ -391,29 +397,31 @@ def test_get_constraints_passing_measurements(self):
         # Constraints:
         #   - Minimum throughput of 100 infer/sec
         #   - Maximum latency of 300 ms
-        constraint_manager = construct_constraint_manager(
-            constraints={
+        constraint_manager = construct_constraint_manager({
+            "profile_models": {
                 "modelA": {
-                    "perf_throughput": {"min": 100},
-                    "perf_latency_p99": {"max": 300}
+                    "constraints": {
+                        "perf_throughput": {"min": 100},
+                        "perf_latency_p99": {"max": 300}
+                    }
                 }
-            }
+            }}
         )
 
         rcm0 = self._construct_rcm(throughput=100, latency=50)  # pass
-        rcm0.set_constraint_manager(constraint_manager=constraint_manager)
+        rcm0.set_constraint_manager(constraint_manager)
```

Comment:
Are these calls needed? It looks like the constraint manager is already set in the function the line above

Reply:
Yes, Tim. As rcm objects are freshly created here, we need to call set_constraint_manager() per rcm object

Reply:
I'm pretty sure that line 411 already handles this

Reply:
ok, yes I see now.